contracts/test/TeleporterTokenBridgeTests.t.sol

```diff
@@ -29,6 +29,12 @@ import {SafeERC20} from "@openzeppelin/contracts@4.8.1/token/ERC20/utils/SafeERC
 abstract contract TeleporterTokenBridgeTest is Test {
     using SafeERC20 for IERC20;
 
+    // convenience struct to reduce stack usage
+    struct OriginSenderInfo {
```

Comment:
Would it make sense to add `sourceBlockchainID` to this struct, and also use it in the contracts themselves?

Reply:
IMO we shouldn't use helper types in interfaces unless they represent something distinct on their own. You could argue that `bridgeAddress`/`senderAddress`/`sourceBlockchainID` represent the source of the Warp message, but without distinct operations performed on that struct, I think they're better left as independent variables. Especially since all they're treated very differently depending on the context.

As for the struct in the test contracts, this was only added to bypass the stack limits, as opposed to representing anything meaningful. I don't particularly care what's contained in it.